StreamChatClient_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -95,12 +95,49 @@ extension NSManagedObjectContext {
 extension ChannelDTO {
     static func channelListFetchRequest(query: ChannelListQuery) -> NSFetchRequest<ChannelDTO> {
         let request = NSFetchRequest<ChannelDTO>(entityName: "ChannelDTO")
-        request.sortDescriptors = [.init(key: "cid", ascending: true)] // TODO: sorting from query
+        request.sortDescriptors = query.sort.sortDescriptors
         request.predicate = NSPredicate(format: "ANY queries.filterHash == %@", query.filter.filterHash)
         return request
     }
 }
 
+public extension Array where Element == Sorting {
```

Comment:
Maybe this should rather live in `Sorting.swift`?

Reply:
Oh, I'm sorry it should be private. I missed that after tests.